photutils/psf/models.py

```diff
@@ -6,6 +6,15 @@
 
 import numpy as np
 
+try:
+    import scipy
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+if HAS_SCIPY:
+    from scipy.interpolate import RectBivariateSpline
+
```

Comment:
@mcara The `try...except` defining `HAS_SCIPY` is used for tests.  I pointing you to that because I thought you had tests that were failing due to the optional `scipy` dependency, but I see that there are no tests in this PR.